crates/rpc/rpc-builder/src/eth.rs

```diff
@@ -1,26 +1,31 @@
+use std::sync::Arc;
+
+use reth_provider::{BlockReaderIdExt, CanonStateSubscriptions, ChainSpecProvider};
```

Comment:
```suggestion
use std::sync::Arc;
use reth_provider::{BlockReaderIdExt, CanonStateSubscriptions, ChainSpecProvider};
```

Reply:
if whitespace is removed then fmt orders alphabetically

Reply:
yes! I love this, very helpful. we want [StdExternalCrate](https://rust-lang.github.io/rustfmt/?version=v1.6.0&search=#StdExternalCrate%5C%3A). feel free to open an issue.